packages/fast-components-react-base/src/tabs/examples.data.tsx

```diff
@@ -1,5 +1,4 @@
 import * as React from "react";
-import { ISnapshotTestSuite } from "@microsoft/fast-jest-snapshots-react";
 import {
     IManagedClasses,
     ITabClassNameContract,
```

Comment:
can we remove the `IManagedClasses` and `ITabClassNameContract` here too?

Reply:
I removed `IManagedClasses` - `ITabClassNameContract` is being used still.